src/client/pages/index.vue

```diff
@@ -23,6 +23,16 @@
           accept="application/json"
           @change="onFileInput">
       </div>
+
+      <div class="page-index__docs-button-wrapper">
+        <md-button
+          :to="`/${$i18n.locale}/documentation/`"
+          :disabled="!acceptedLegal"
+          class="md-raised page-index__docs-button">
```

Comment:
What about a non raised button? IMHO it is still a very large button although it is not a primary button

Reply:
I prefer a btn with a box layout, so the whole popup doesn't look too cluttered. Now the two levels (blue btns, white btn) are aligned => calm. Plus the docs btn should somehow be well visible